pipelinewise/cli/__init__.py

```diff
@@ -19,7 +19,8 @@
 
 __version__ = get_distribution('pipelinewise').version
 USER_HOME = os.path.expanduser('~')
-CONFIG_DIR = os.path.join(USER_HOME, '.pipelinewise')
+DEFAULT_CONFIG_DIR = os.path.join(USER_HOME, '.pipelinewise')
+CONFIG_DIR = os.environ.get('PIPELINEWISE_CONFIG', DEFAULT_CONFIG_DIR)
```

Comment:
Suggest we name this `PIPELINEWISE_CONFIG_DIRECTORY` to be clear this is not a file path (which could be inferred from the current name).